File: src/test/java/tests/de/espend/idea/vuejs/mcp/McpCollectorTestCase.java
package tests.de.espend.idea.vuejs.mcp;

import tests.de.espend.idea.vuejs.VueJsLightJavaCodeInsightFixtureTestCase;

public abstract class McpCollectorTestCase extends VueJsLightJavaCodeInsightFixtureTestCase {
    @Override
    public String getTestDataPath() {
        return "src/test/java/tests/de/espend/idea/vuejs/mcp/fixtures";
    }
}

File: src/test/kotlin/tests/de/espend/idea/vuejs/mcp/McpCollectorTestCase.kt
package tests.de.espend.idea.vuejs.mcp

import tests.de.espend.idea.vuejs.VueJsLightJavaCodeInsightFixtureTestCase

abstract class McpCollectorTestCase : VueJsLightJavaCodeInsightFixtureTestCase() {
    override fun getTestDataPath() = "src/test/kotlin/tests/de/espend/idea/vuejs/mcp/fixtures"
}
